app/models/User.coffee

```diff
@@ -267,6 +269,15 @@ module.exports = class User extends CocoModel
     myHeroClasses.push heroClass for heroClass, heroSlugs of ThangTypeConstants.heroClasses when _.intersection(myHeroSlugs, heroSlugs).length
     myHeroClasses
 
+  getHeroPoseImage: co.wrap ->
+    options={}
+    options.method = 'GET'
+    options.contentType = 'application/json'
+    options.dataType = 'json'
+    if ((@get('heroConfig') or {}).thangType)
+      poseFile = yield fetchJson("/db/thang.type/#{((@get('heroConfig') or {}).thangType)}/version",options)
+      $("#hero-pose").attr('src', '/file/' + poseFile.poseImage)
```

Comment:
This should just return the URL and leave it up to the caller what to do with it
```suggestion
      return '/file/' + poseFile.poseImage
```